sonar-python-plugin/src/main/java/org/sonar/plugins/python/PythonSquidSensor.java

```diff
@@ -93,9 +100,48 @@ public void analyse(Project project, SensorContext context) {
     scanner.scanFiles(Lists.newArrayList(fileSystem.files(p.and(p.hasType(InputFile.Type.MAIN), p.hasLanguage(Python.KEY)))));
 
     Collection<SourceCode> squidSourceFiles = scanner.getIndex().search(new QueryByType(SourceFile.class));
+    savePreciseIssues(
+      visitors
+        .stream()
+        .filter(v -> v instanceof PythonCheck)
+        .map(v -> (PythonCheck) v)
+        .collect(Collectors.toList()),
+      context);
     save(squidSourceFiles);
   }
 
+  // fixme (Lena): unit test it with new Sensor API
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Take the required action to fix the issue indicated by this comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1134)